onnxruntime/test/providers/cpu/reduction/reduction_ops_test.cc

```diff
@@ -760,7 +771,12 @@ TEST(ReductionOpTest, ReduceMean_int32) {
                           90, 100,
                           110, 120});
   test.AddOutput<int32_t>("reduced", {1, 2, 1}, {55, 75});
+  //Disbaled temporarily for OpenVINO
```

Comment:
typo Disabled 

some more details on why?

Reply:
No longer disabled. Removed this condition